### DISCUSSION TITLE
My project works properly on Android and it misses some planes on IOS

### DISCUSSION THREAD

**Discussion by MahdiHabibiXr** [google/model-viewer#4337] [General]
Hey guys, 
I'm making an AR project for one of my 3d models.
I made this 3d model using 3DMax and convert it to glb file using Blender.
On Android everything is fine, In the scene viewer and even in AR mode, But when i open my project on IOS and open it in AR, Some parts of my 3d model is not showing!
Here is the link to my project live page : https://denitte.com/ar/phone1/
![image](https://github.com/google/model-viewer/assets/97017176/b47ef45d-2403-4fcb-a526-23121a345f05)
![image](https://github.com/google/model-viewer/assets/97017176/bd2e8074-b6f3-44a4-8fe0-d9d350a66cca)
Like the roof or the sofa. This is not happening in Android ... Why is that? Thanks.

**Comment by elalish** [comment]
Most likely is an issue with the three.js USDZ auto-generation. You can try making your own USDZ and supplying it with `ios-src`. @meshula in case it's a QuickLook issue.
